lib/services/datafactoryManagement/lib/models/blobTrigger.js

```diff
@@ -20,14 +20,14 @@ const models = require('./index');
 class BlobTrigger extends models['MultiplePipelineTrigger'] {
   /**
    * Create a BlobTrigger.
-   * @member {string} [folderPath] The path of the container/folder that will
+   * @member {string} folderPath The path of the container/folder that will
```

Comment:
Making an optional parameter required is a breaking change. Requires major version increment.